templates/devspace.yaml.tpl

```diff
@@ -94,6 +94,13 @@ vars:
 deployments:
   app:
     namespace: ${DEVENV_DEPLOY_NAMESPACE}
+    resources:
+      requests:
+        cpu: 1
+        memory: 1Gi
+      limits:
+        cpu: 5
+        memory: 10Gi
```

Comment:
This was a pretty arbitrary set of values, we can change it if it causes problems.